robot/EDA/tests/browser/regression/run_regression_suite1.robot

```diff
@@ -2,32 +2,33 @@
 
 Resource        robot/EDA/resources/EDA.robot
 Resource        robot/EDA/resources/test_data.robot
-
 Suite Setup     Run keywords
 ...             Open Test Browser
 ...             Initialize Test Data
-
 Suite Teardown  Delete Records and Close Browser
 
 *** Test Cases ***
-    
 Create Contact And Verify
-    ${contact_id} =  Populate Create And Return Contact with Address    Joe      Mazzocco   5345 Calero Ave             San Jose     95023     CA     USA    
-    &{contact} =  Salesforce Get    Contact             ${contact_id}
-    Header Field Value              Account Name        &{contact}[LastName] Administrative Account
-    Select Tab    Details 
-    Page Should Contain    5345 Calero Ave  
-    Go To Object Home         Contact
-    Verify Record    &{contact}[FirstName] &{contact}[LastName]
+    [tags]                                      unstable
```

Comment:
@Safiyab @dashattuck  Just curious, do we have a plan for how to resolve unstable robot tests? I noticed that all of our regression tests here are now marked as unstable...

Reply:
@robertbwatson We should ideally be tracking the unstable tests to figure out a pattern of their failure and then fix them accordingly. Coz it is not always clear why they fail, some reasons being slow servers, element taking too long to load, screen size, overlapping elements etc.

I did look into some of these unstable tests and came up with some solutions, but I want to check with Doug before I make these changes (I want to be sure I am not missing something he already thought of).

I created a WI on our automation board a few mins ago ([W-034924](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001C8ahQAC/view)) and we will pick this up once Doug is back.

Reply:
@Safiyab Thank you for creating the work item!